src/cljc/athens/common_events.cljc

```diff
@@ -206,14 +206,14 @@
 (defn build-unindent-multi-event
   "Builds `:datascript/unindent-multi` event with:
   - `uids` : `:block/uid` of selected blocks
-  - `f-uid`: first uid in uids that is passed through `uid-and-embed-id` function"
+  - `first-uid`: first uid in uids that is passed through `uid-and-embed-id` function"
```

Comment:
Maybe `first-uid` is good name, but the description could be more well descriptive.
Currently it only mentions some current implementation detail, but this piece of data (`first-uid`) has a meaning.
What meaning is that? -> Ask why do we need it during resolution, what role does it play there?